src/migrations/data.ts

```diff
@@ -19,7 +19,7 @@ import { migrationStepRegistry } from "./migration_steps";
  * a breaking change is made in the way the state is handled, and an upgrade
  * function should be defined
  */
-export const CURRENT_VERSION = 20;
+export const CURRENT_VERSION = 21;
```

Comment:
we are now at version 21 already, it should be 22 here :sweat_smile: :rocket:

Reply:
Rebased :)